.gitlab-ci.yml

```diff
@@ -33,11 +33,18 @@ after_script:
   - stopdocker || true
 
 stages:
+  - docs
```

Comment:
Can this be in parallel with the `linting` step?

Reply:
Yep, just added it in. Thanks for the suggestion!